tenant/sunbird/learnathon.html

```diff
@@ -338,7 +338,7 @@ <h4 class="abt-sec-title text-center title-top-margin">Solution submission</h4>
         <div class="num-cont1 text-white">First Phase Results</div>
       </div>
       <div>
-        <div class="number1 text-white">22nd June</div>
+        <div class="number1 text-white">12th June</div>
```

Comment:
@shreyas1434shinde Why changed the Date?

Reply:
@komalm date changes reverted